packages/zeplin-extension-style-kit/package.json

```diff
@@ -5,7 +5,9 @@
   "main": "index.js",
   "scripts": {
     "publish-pkg": "./scripts/publish.sh",
-    "build": "babel ./ -d ./lib --ignore tests,scripts,node_modules,package.json --copy-files",
+    "prebuild": "npm run clean",
+    "build": "babel ./ -d ./lib --ignore lib,tests,scripts,node_modules --copy-files",
```

Comment:
with Babel 7 the behavior of --copy-files flag is changed. So we need to remove scripts, tests, node_modules folder manually